source/TLS.rst

```diff
@@ -0,0 +1,182 @@
+
+* ``spec.sslCA`` key should contain the name of the secret with TLS
+  :abbr:`CA (Certificate authority)` used for both connection encryption
+  (external traffic), and replication (internal traffic),
+* ``spec.sslSecretName`` key should contain the name of the secret created to
+  encrypt **external** communications,
+* ``spec.secrets.sslReplicationSecretName`` key should contain the name of the
+  secret created to encrypt **internal** communications,
+* ``spec.tlsOnly`` key should be set to ``true`` if you want to disable
+  unencrypted communications.
+
+Don't forget to apply changes as usual:
+
+.. code:: bash
+
+   $ kubectl apply -f deploy/cr.yaml
+
+Check connectivity to the cluster
+---------------------------------
+
+You can check TLS communication with use of the ``psql``, the standart interactive terminal-based front-end to PostgreSQL. The following command will spawn a new ``pg-client`` container, which includes needed command and can be used for the check (use your real cluster name instead of the ``<cluster-name>`` placeholder):
+
+.. code:: bash
+
+   $ cat <<EOF | kubectl apply -f -
+   apiVersion: apps/v1
+   kind: Deployment
+   metadata:
+     name: pg-client
+   spec:
+     replicas: 1
+     selector:
+       matchLabels:
+         name: pg-client
+     template:
+       metadata:
+         labels:
+           name: pg-client
+       spec:
+         containers:
+           - name: pg-client
+             image: perconalab/percona-distribution-postgresql:13.2
+             imagePullPolicy: Always
+             command:
+             - sleep
+             args:
+             - "100500"
+             volumeMounts:
+               - name: ca
+                 mountPath: "/tmp/tls"
+         volumes:
+         - name: ca
+           secret:
+             secretName: <cluster_name>-ssl-ca
+             items:
+             - key: ca.crt
+               path: ca.crt
+               mode: 0777
+   EOF
+
+Now get shell access to the newly created container, and launch the PostgreSQL interactive terminal to check connectivity over the encrypted channel (please use real cluster-name, PostgreSQL user login and password):
+
+.. code:: bash
+
+   $ kubectl exec -it deployment/pg-client -- bash -il
+   [postgres@pg-client /]$ PGSSLMODE=verify-ca PGSSLROOTCERT=/tmp/tls/ca.crt psql postgres://<postgresql-user>:<postgresql-password>@<cluster-name>-pgbouncer.<namespace>.svc.cluster.local
+
+Now you should see the prompt of PostgreSQL interactive terminal:
+
+.. code:: bash
+
+   psql (13.2)
+   Type "help" for help.
+   pgdb=>
+
+Run Percona Distribution for PostgreSQL without TLS
+===================================================
+
+Omitting TLS is also possible, but we recommend that you run your cluster with the TLS protocol enabled.
+
+To disable TLS protocol (e.g. for demonstration purposes) set the
+``spec.allowUnsafeConfigurations`` key to ``true`` and ``spec.tlsOnly`` key to
```

Comment:
no such thing like allowUnsafeConfigurations. Only tlsOnly and ssl secret name options